src/components/cms/SkillCreator/SkillWizard.js

```diff
@@ -1120,43 +1125,51 @@ class SkillWizard extends Component {
                 />
                 {accessToken && this.state.editable && (
                   <PaperContainer>
-                    <InfoIcon
-                      data-tip={`Learn more about <a href=${urls.GITHUB_URL +
-                        '/blob/master/docs/Skill_Tutorial.md'} rel="noopener noreferrer" target="_blank" >SUSI Skill Language</a>`}
-                    />
+                    <a
+                      href="https://github.com/fossasia/susi.ai/blob/master/docs/Skill_Tutorial.md"
+                      rel="noopener noreferrer"
+                      target="_blank"
+                    >
+                      <InfoIcon data-tip="Learn more about SUSI Skill Language" />
+                    </a>
                     <CenterDiv>
                       <DropDownDiv>
                         <DropDownWrap>
                           <SkillDetail>
                             <DetailText>Category:&nbsp;</DetailText>
-                            <SelectDropDown
+                            <OutlinedSelectField
+                              select
                               value={category}
                               onChange={this.handleGroupChange}
                               autoWidth={true}
+                              variant="standard"
                             >
                               {this.state.groups}
-                            </SelectDropDown>
+                            </OutlinedSelectField>
                           </SkillDetail>
                           <SkillDetail>
                             <DetailText>Language:&nbsp;</DetailText>
-                            <SelectDropDown
+                            <OutlinedSelectField
+                              select
                               value={language}
                               onChange={this.handleLanguageChange}
                               autoWidth={true}
+                              variant="standard"
                             >
                               {this.state.languages}
-                            </SelectDropDown>
+                            </OutlinedSelectField>
                           </SkillDetail>
                           <SkillDetail>
                             <DetailText>
                               {this.isBotBuilder
                                 ? 'Bot Name: '
                                 : 'Skill Name: '}
                             </DetailText>
-                            <NameField
+                            <OutlinedInput
                               margin="normal"
                               value={name}
                               onChange={this.handleExpertChange}
+                              variant="standard"
```

Comment:
Change this to `variant="outlined"`

For outlined select look at this : https://material-ui.com/components/selects/